fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -196,7 +196,8 @@ public class PropertyAnalyzer {
      */
     public static final String PROPERTY_MV_SORT_KEYS = "mv_sort_keys";
 
-    // light schema change
+    // fast schema evolution
+    public static final String PROPERTIES_USE_FAST_SCHEMA_EVOLUTION = "fast_schema_evolution";
     public static final String PROPERTIES_USE_LIGHT_SCHEMA_CHANGE = "light_schema_change";
```

Comment:
remove?

Reply:
> remove?

I keep `light_schema_change` property because our admin test is `light_schema_change` right now. If we remove it directly, we need to modify admin test first, but modify directly will affect the other prs, so I  keep it in this pr and after merged, I will modify admin test and remove it.